### ISSUE TITLE
Mobile: wrong word in lookup does not produce any error message

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios_nemo_ui#122]
tested in 1.0.0.7 on iPhone 6s. I enter a word such as fyhd in the "new" lookup and got an empty morphology pop-up. in the alpheios tool and text environment on desktop I get the "lexical query produced no results" message and then the word itself with no definition and the credits on the bottom.

**Comment by balmas** [comment]
yeah, I noticed this too. I think maybe notifications are missing from the mobile morphology component. Assigning to @kirlat
